gravitee-apim-cypress/cypress/fixtures/json/imports/pages/api-with-documentation.json

```diff
@@ -0,0 +1,134 @@
+{
```

Comment:
I would have prefered another path, maybe:
`fixtures/json/api/imports/api-with-documentation.json`

wdyt ?

Reply:
Yes. We removed fixtures files at all, in favor of fakers.